src/aws-resource-mappings.ts

```diff
@@ -444,7 +444,6 @@ export function mapToAwsResource(
             const policy = new aws.iam.Policy(
                 logicalId,
                 {
-                    name: rawProps.PolicyName,
```

Comment:
CloudFormation requires that a name be set so CDK sets a name automatically, but if not set then pulumi will auto name it which is better.

This fixes some inter-test conflicts.